indico/htdocs/js/presentation/Data/Remote.js

```diff
@@ -17,51 +12,51 @@ var ReadyState = new Enum("Uninitialized", "Loading", "Loaded", "Interactive", "
  * @return {WatchAccessor} value
  */
 function jsonRpcValue(url, method, params, def, dontStart, callback) {
-        // not an object for url... curry
-        var self = new Source();
-        var object = mixWatchGetters(self, ["state", "error"]);
-        object.set("data", def);
-        mixinInstance(self, object.accessor("data"), WatchAccessor);
-
-        function process(result, error) {
-                if (exists(error)) {
-                        object.update({
-                                state: SourceState.Error,
-                                error: error
-                        });
-                } else {
-                        object.update({
-                                state: SourceState.Loaded,
-                                data: result,
-                                error: null
-                        });
+    // not an object for url... curry
+    var self = new Source();
+    var object = mixWatchGetters(self, ["state", "error"]);
+    object.set("data", def);
+    mixinInstance(self, object.accessor("data"), WatchAccessor);
 
-                        if(callback) {
-                            callback();
-                        }
-                }
+    function process(result, error) {
+        if (exists(error)) {
+            object.update({
+                state: SourceState.Error,
+                error: error
+            });
+        } else {
+            object.update({
+                state: SourceState.Loaded,
+                data: result,
+                error: null
+            });
         }
 
-        self.set = function(value) {
-            var old = self.get();
-            object.update({
-                        state: SourceState.Committing,
-                        error: null
-                });
-            jsonRpcCommit(url, method, params, value, process);
-            return old;
-        };
-        self.refresh = function() {
-                object.update({
-                        state: SourceState.Loading,
-                        error: null
-                });
-                return jsonRpc(url, method, params, process);
-        };
-        if (!dontStart) {
-            self.refresh();
+        if (callback) {
```

Comment:
`$.isFunction(callback)`?

Reply:
that's legacy code, and we can assume a `callback` is callable anyway ;)